Code/GraphMol/Substruct/catch_tests.cpp

```diff
@@ -15,11 +17,49 @@
 
 #include <GraphMol/RDKitBase.h>
 #include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/SmilesParse/SmilesWrite.h>
 #include <GraphMol/Substruct/SubstructMatch.h>
 
 using namespace RDKit;
 typedef std::tuple<std::string, std::string, size_t> matchCase;
 
+class _IsSubstructOf : public Catch::MatcherBase<const ROMol &> {
+  ROMol const *m_mol;
+  SubstructMatchParameters m_ps;
+
+ public:
+  _IsSubstructOf(const ROMol &m) : m_mol(&m) {}
+
+  _IsSubstructOf(const ROMol &m, SubstructMatchParameters ps)
+      : m_mol(&m), m_ps(ps) {}
+
+  virtual bool match(const ROMol &query) const override {
+    return !SubstructMatch(*m_mol, query, m_ps).empty();
+  }
+
+  virtual std::string describe() const override {
+    std::ostringstream ss;
+    ss << "is not a substructure of " << MolToCXSmiles(*m_mol);
+    return ss.str();
+  }
+};
+
+static _IsSubstructOf IsSubstructOf(const ROMol &m,
+                                    const SubstructMatchParameters &ps) {
+  return _IsSubstructOf(m, ps);
+}
+
+static _IsSubstructOf IsSubstructOf(const ROMol &m) {
+  return _IsSubstructOf(m);
+}
+
+namespace Catch {
+template <>
+struct StringMaker<RDKit::RWMol> {
```

Comment:
Why `RWMol` here?

Reply:
`""_smiles` gives a `unique_ptr<RWMol>` so I was just blindly pasting. updating to `ROMol`.

Reply:
requires an RWMol to get correct printing.